subpackages/assertions/source/unit_threaded/assertions.d

```diff
@@ -480,7 +480,7 @@ private auto threw(T : Throwable, E)(lazy E expr)
 
 
 // Formats output in different lines
-private string[] formatValueInItsOwnLine(T)(in string prefix, scope auto ref T value) {
```

Comment:
Why remove `in`?

Reply:
`in` doesn't imply `scope` without `-preview=in`. I didn't want to add `-preview=in` to unit-threaded because that's an unapproved feature.

Reply:
> Error: attribute `scope` cannot be applied with `in`, use `-preview=in` instead

Reply:
> Why remove `in`? 

I asked the same question. See above.